service/svc.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+ROOTPROJECTPATH="$(
+    cd -- "$(dirname "$0")/.." >/dev/null 2>&1
+    pwd -P
+)"
+
+echo $ROOTPROJECTPATH
+
+function install {
+    path=$1
+    if [ -z "$1" ]; then
+        path="/lib/systemd/system"
+    fi
+
+    if [ -e "$path/threeal-bot.service" ]; then
+        echo "Service is already installed"
+        exit
+    fi
+
+    workdir=$ROOTPROJECTPATH
+
+    execstart="go run $workdir/cmd/backend/main.go"
+
+    echo "Installing service..."
+
+    sed -e "s@<user>@$USER@" -e "s@<group>@$USER@" -e "s@<workdir>@$workdir@" -e "s@<pathexists>@$workdir@" -e "s@<execstart>@$execstart@" service/threeal-bot.service >threeal-bot.service
```

Comment:
Just use `<workdir>` for setting `ConditionPathExists`, `WorkingDirectory`, and `ExecStart`. You can achieve that by replacing `s` prefix with `g` (global, replace all).

Let's do the same with `<user>` and `<group>`  if both are using `$USER`.